Jenkinsfile

```diff
@@ -1,7 +1,8 @@
-// Builds the plugin using https://github.com/jenkins-infra/pipeline-library
-buildPlugin(useContainerAgent: true, configurations: [
-  // Test the minimum required Jenkins Version.
-  [ platform: 'linux', jdk: '11'],
-  [ platform: 'windows', jdk: '17'],
-  [ platform: 'linux', jdk: '21']
+/*
+ See the documentation for more options:
+ https://github.com/jenkins-infra/pipeline-library/
+*/
+buildPlugin(useContainerAgent: false, configurations: [
+  [platform: 'linux', jdk: 21],
```

Comment:
2.479 only drop support for Java 11. Not 17. 

Might be useful to keep Java 17 build ? (See for the archetype https://github.com/jenkinsci/archetypes/blob/master/common-files/Jenkinsfile)

Reply:
Thanks! https://github.com/jenkinsci/opentelemetry-api-plugin/pull/20